### PR TITLE
Fixed few typos in the Symbol tutorial

### PR DESCRIPTION
**Pull request by apaleyes**
Spotted few typos as I was going through the Symbol tutorial

### CODE REVIEW

File: docs/tutorials/basic/symbol.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ composed.list_arguments()
 ```
 
 In this example, *net2* is used as a function to apply to an existing symbol *net1*,
-and the resulting *composed_net* will have all the attributes of *net1* and *net2*.
+and the resulting *composed* will have all the attributes of *net1* and *net2*.
```
Comment (piiswrong) [inline docs/tutorials/basic/symbol.md]: I'm not sure composed is better here. Maybe *composed* symbol

Comment (apaleyes) [inline docs/tutorials/basic/symbol.md]: _composed_ is just in line with the actual code sample, and I felt the sample should take priority. But we can change example for sure. I like _composed_net_ the most. Sounds ok, or do you still prefer _composed_symbol_?

Comment (piiswrong) [inline docs/tutorials/basic/symbol.md]: I meant `*composed* symbol`, meaning a symbol called composed

Comment (apaleyes) [inline docs/tutorials/basic/symbol.md]: Oh, I see what you mean now. That makes sense, updated

**Pull request merged**